developerTools/css/brandColors.css

```diff
@@ -0,0 +1,76 @@
+.hifi-red-text {
+    color: #ff001a;   
+}
+.hifi-red-background {
```

Comment:
looks like there are no -background and -text styles that are different. why not just use, for example, `.hifi-red` ?